### ISSUE TITLE
move over other help file

### ISSUE THREAD

**Issue by lshep** [Bioconductor/pkgrevdocs#54]
http://bioconductor.org/developers/how-to/buildingPackagesForBioc/

referenced in conclusion -- is this still relevant and if it is move it over

**Comment by kevinrue** [comment]
Hm... That looks like a lot of good pieces for the mentorship program (https://github.com/Bioconductor/bioc_mentorship_docs)

Although, as mentioned in other issues, the line between pkgrevdocs and bioc_mentorship_docs is tricky.

I think that linked page has a lot of relevant contents, and the challenge would be to move it piece by piece in either pkgrevdocs or bioc_mentorship_docs, while deleting it from this page on the bioconductor website (if that is the objective).

**Comment by lshep** [comment]
yes I think the objective is to get rid of it on bioconductor.org and move it to one of these other locations.  esp since its hidden on the bioconductor.org website too.

**Comment by lshep** [comment]
git show 31dd7d89929ae2c604e4cfd411065e34c7cb1c5e
